examples/query_elasticsearch/README.md

```diff
@@ -53,7 +53,32 @@ python3 elasticsearch_pageviews.py --config <config_file> --username $ES_USER --
 A sample result will appear as (the following is not real data):
 ```
 2019-07-15 14:54:40,181:[INFO]: intranet.gov.bc.ca page views successfully queried
-Domain:  intranet.gov.bc.ca  Page Views:  4260
+Domain:  intranet.gov.bc.ca  Page Views:  4230
 2019-07-15 14:54:40,362:[INFO]: www.env.gov.bc.ca page views successfully queried
-Domain:  www.env.gov.bc.ca  Page Views:  4154
+Domain:  www.env.gov.bc.ca  Page Views:  4554
+```
+
+### [elasticsearch_queuelength.py](./elasticsearch_queuelength.py)
+Queries elasticsearch for the current day and calculates the queue length for given Service BC offices.
+
+Configuration file should contain line separated list of Service BC office locations (eg: Kelowna, Kamloops) to query from Elasticsearch index and endpoint specified.
+
+A file containing the json export of a Looker explore is required in order to search by Service BC office names. This can be exported from the following Explore: https://analytics.gov.bc.ca/explore/cfms_poc/cfms_poc?toggle=fil&qid=DWe1cjJI5lYB084l8fjZ1M
```

Comment:
Since this is a public facing README and this link will not be accessible to anyone outside of ourselves, this sort of documentation should be on an internal document like the JIRA ticket or in Confluence.

Reply:
Removed from readme and added as a comment to the ticket